.travis.yml

```diff
@@ -10,6 +10,10 @@ before_script:
   - cp keys/saml.key.enc.example keys/saml.key.enc
   - bin/rake db:setup --trace
   - bin/rake assets:precompile
+after_script:
+  - npm install -g pa11y-crawl
+  - bundle exec rails s -d
+  - pa11y-crawl --standard Section508 --ci http://localhost:3000
```

Comment:
Yes, the server is ready when the crawl starts. 
- https://continua11y.18f.gov/18F/identity-idp/7f5dd0c37fb1d3e70036575286a862a1b0e1dfef